style.css

```diff
@@ -75,3 +75,360 @@
   margin-bottom: 120px;
```

Comment:
[OPTIONAL]
Work professionally in your future projects and improve your CSS code using DRY method.

Reply:
Thanks a lot, Carlos!
I'm happy to read you, and of course, I will use DRY method. 🤓 

Also, I'm from Xalapa, Veracruz, it's nice to see Mexicans 😃